modules/stanford_news_views/stanford_news_views.views_default.inc

```diff
@@ -1264,7 +1269,7 @@ function stanford_news_views_views_default_views() {
   );
   $handler->display->display_options['filters']['field_s_news_categories_tid']['type'] = 'select';
   $handler->display->display_options['filters']['field_s_news_categories_tid']['vocabulary'] = 'news_categories';
-  $handler->display->display_options['path'] = 'news/recent-news';
+  $handler->display->display_options['path'] = 'news/in-the-news';
```

Comment:
Ah I see the mock up on the ticket now. This is a new page. I will add it to the deploy.